### ISSUE TITLE
EPERM on Windows for Nodemon

### ISSUE THREAD

**Issue by serganus** [remy/nodemon#608]
```
C:\>node C:\Users\serga\AppData\Roaming\npm\node_modules\nodemon\bin\nodemon.js ap
p.js --dump
exception in nodemon killing node
Error: watch EPERM
    at exports._errnoException (util.js:746:11)
    at FSWatcher.start (fs.js:1172:11)
    at Object.fs.watch (fs.js:1198:11)
    at Function.check (C:\Users\serga\AppData\Roaming\npm\node_modules\nodemon\l
ib\config\watchable.js:39:6)
    at alternativeCheck (C:\Users\serga\AppData\Roaming\npm\node_modules\nodemon
\lib\config\checkWatchSupport.js:33:15)
    at checkWatchSupport (C:\Users\serga\AppData\Roaming\npm\node_modules\nodemo
n\lib\config\checkWatchSupport.js:65:5)
    at C:\Users\serga\AppData\Roaming\npm\node_modules\nodemon\lib\config\index.
js:71:5
    at normaliseRules (C:\Users\serga\AppData\Roaming\npm\node_modules\nodemon\l
ib\config\load.js:139:3)
    at ready (C:\Users\serga\AppData\Roaming\npm\node_modules\nodemon\lib\config
\load.js:80:9)
    at C:\Users\serga\AppData\Roaming\npm\node_modules\nodemon\lib\config\load.j
s:92:13

----------------------------------------------------------
If appropriate, please file an error with the output from:
$ node C:\Users\serga\AppData\Roaming\npm\node_modules\nodemon\bin\nodemon.js ap
p.js --dump
At http://github.com/remy/nodemon/issues/new
----------------------------------------------------------

12 Aug 10:54:06 - [nodemon] v1.4.0
12 Aug 10:54:06 - [nodemon] to restart at any time, enter `rs`
12 Aug 10:54:06 - [nodemon] watching: *.*
--------------
node: v0.12.7
nodemon: v1.4.0
command: node C:\Users\serga\AppData\Roaming\npm\node_modules\nodemon\bin\nodemo
n.js app.js --dump
cwd: C:\
OS: win32 x64
--------------
{ run: false,
  system:
   { cwd: 'C:\\',
     useFind: false,
     useWatch: false,
     useWatchFile: true },
  required: false,
  dirs: [ 'C:\\' ],
  timeout: 1000,
  options:
   { dump: true,
     ignore:
      [ '.git',
        'node_modules',
        'bower_components',
        '.sass-cache',
        re: /\.git|node_modules|bower_components|\.sass\-cache/ ],
     watch: [ '*.*', re: /.*\..*/ ],
     restartable: 'rs',
     execMap: { py: 'python', rb: 'ruby' },
     stdin: true,
     runOnChangeOnly: false,
     verbose: false,
     stdout: true,
     execOptions:
      { script: 'app.js',
        exec: 'node',
        args: [],
        scriptPosition: 0,
        nodeArgs: undefined,
        ext: 'js',
        env: {},
        execArgs: [] },
     monitor:
      [ '*.*',
        '!C:\\.git/**/*',
        '!C:\\node_modules/**/*',
        '!C:\\bower_components/**/*',
        '!.sass-cache' ] },
  load: [Function],
  reset: [Function: reset],
  lastStarted: 0,
  loaded: [],
  command:
   { raw: { executable: 'node', args: [ 'app.js' ] },
     string: 'node app.js' },
  offset: 0 }
--------------
```

**Comment by remy** [comment]
Please try the build here #625 and feedback against the issue #625.

**Comment by remy** [comment]
#625 closes and released in `nodemon@1.4.2`
